resources/developer-mode.mjs

```diff
@@ -222,14 +222,20 @@ function createUIForSuites() {
 }
 
 function createUIForRun() {
-    let button = document.createElement("button");
-    button.textContent = "Start Test";
-    button.onclick = (event) => {
+    const stepTestButton = document.createElement("button");
+    stepTestButton.textContent = "Step Test ⏯";
```

Comment:
Can we use `&#9199;` or '\u23EF' instead of directly using the unicode character?

Reply:
done.